### PR TITLE
Add Ray Serve to README.rst

### PR DESCRIPTION
**Pull request by simon-mo**
## Why are these changes needed?
This PR adds Ray Serve examples to the readme quickstart.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Comment by simon-mo** [comment]
Rendered at https://github.com/simon-mo/ray/tree/add-serve-to-doc#ray-serve-quick-start

**Review by bllchmbrs — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -182,6 +182,104 @@ RLlib Quick Start
...
+    model = GradientBoostingClassifier()
+    model.fit(data[:100], target[:100])
+
+    # Save the model and label to file
+    with open("/tmp/iris_model.pkl", "wb") as f:
+        pickle.dump({"model": model, "labels": target_names.tolist()}, f)
+
+    # Define Ray Serve model,
+    class BoostingModel:
+        def __init__(self):
```
Comment (richardliaw) [inline README.rst:243]: can you shorten this somehow?

Comment (richardliaw) [reply README.rst:243]: the example seems quite long

File: README.rst [hunk 2]
```diff
@@ -182,6 +182,104 @@ RLlib Quick Start
 .. _`RLlib`: https://docs.ray.io/en/latest/rllib.html
 
 
+Ray Serve Quick Start
+---------------------
+
```
Comment (richardliaw) [inline README.rst:187]: be sure to add to toc at top of readme??

Comment (simon-mo) [reply README.rst:188]: aha missed that. thanks!

**Comment by robertnishihara** [comment]
The logo is too big!

**Pull request merged**